.github/workflows/transports-ci.yml

```
@@ -0,0 +1,200 @@
+      - name: Build UI static files
+        working-directory: ui
+        run: |
+          npm install
+          npm run build
+
+      # Cross-compile Go binaries for multiple platforms
+      # Uses existing script that builds for Windows, macOS, Linux (multiple architectures)
+      # Output goes to ../dist/apps/bifrost for upload script
+      - name: Build Go executables
+        working-directory: transports
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2046:warning:3:87: Quote this to prevent word splitting

(shellcheck)